templates/glue-workflow.j2

```diff
@@ -300,6 +308,34 @@ Resources:
       StartOnCreation: true
       WorkflowName: !Ref JsonToParquetWorkflow
 
+
+  {% for dataset in datasets if dataset["data_type"].lower() in sceptre_user_data.data_values_expectations %}
+  {{ dataset['stackname_prefix'] }}GreatExpectationsParquetTrigger:
+    Type: AWS::Glue::Trigger
+    Properties:
+      Name: !Sub "${Namespace}-{{ dataset['stackname_prefix'] }}GreatExpectationsParquetTrigger"
+      Actions:
+        - JobName: !Sub ${Namespace}-{{ dataset["stackname_prefix"] }}-GreatExpectationsParquetJob
+          Arguments:
+            "--data-type": {{ dataset["data_type"].lower() }}
+            "--namespace": !Ref Namespace
+            "--cfn-bucket": !Ref CloudformationBucketName
+            "--parquet-bucket": !Ref ParquetBucketName
+            "--shareable-artifacts-bucket": !Ref ShareableArtifactsBucketName
+            "--expectation-suite-key-prefix": !Sub "${Namespace}/src/glue/resources/data_values_expectations.json"
+            "--additional-python-modules": "great_expectations~=0.18,urllib3<2"
+      Description:  This trigger runs the great expectation parquet job for this data type after completion of the JSON to Parquet job for this data type
+      Type: CONDITIONAL
+      Predicate:
+        Conditions:
+          - JobName: !Sub "${Namespace}-{{ dataset['stackname_prefix'] }}-Job"
+            State: SUCCEEDED
+            LogicalOperator: EQUALS
+        Logical: AND
```

Comment:
This is optional if there is only one condition https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-trigger-predicate.html